Policies/github-acceptable-use-policies.md

```diff
@@ -31,7 +31,9 @@ Under no circumstances will Users upload, post, host, execute, or transmit any C
 
 - is or contains false, inaccurate, or intentionally deceptive information that is likely to adversely affect the public interest (including health, safety, election integrity, and civic participation);
 
-- contains or installs any active malware or exploits, or uses our platform for exploit delivery (such as part of a command and control system); or
+- contains or installs any active malware or exploits, or uses our platform for exploit delivery (such as part of a command and control system); 
+
+- shares unauthorized product licensing keys, software for generating unauthorized product licensing keys, or software for bypassing checks for product licensing keys, including extension of a free license beyond its trial period; or
```

Comment:
I really don't think random number generators should be added to the DMCA policy.

In any case, https://github.com/github/site-policy/pull/400

Reply:
Thanks. They aren't. :+1: